com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -405,23 +397,32 @@ RosettaMapPathValue:
  */ 
  
 RosettaDocReference:
-	'[' ('regulatoryReference'|'docReference') body=[RosettaBody|ValidID] corpuses+=([RosettaCorpus|ValidID])+ (segments+=RosettaSegmentRef)+ ('provision' provision=STRING)? (reportedField?='reportedField')?  ']'
+	'[' ('regulatoryReference'|'docReference') docReference=RegulatoryDocumentReference
+		(rationales+=DocumentRationale)*
+		// @Compat. In DRR, there is a segment `table` located after a `rationale`. This should never be the case, but to remain backwards compatible, we need to allow this.
+ 		// NOTE: once we get rid of `extraneousSegments`, we can also remove the syntactic predicate `=>` on the segment of rule `RegulatoryDocumentReference`.
+ 		(extraneousSegments+=RosettaSegmentRef)*
+		('structured_provision' structuredProvision=STRING)?
+		('provision' provision=STRING)?
+		(reportedField?='reportedField')? ']'
```

Comment:
Cleanup 4: `rationale`, `rationale_author` and `structured_provision` are now actual keywords instead of document segments.